### PR TITLE
html and css added to home-page

### LINKED ISSUE #1: home page view
- [X] HTML
- [X] CSS

### PR DESCRIPTION
**Pull request by tahminarasoli**
This will close #1

### CODE REVIEW

**Comment by tahminarasoli** [comment]
@22count22 Patrick please have a look and write your comment if there is any if not then @bermarte is accountable to merge

File: index.html [hunk 1]
```diff
@@ -1,25 +1,47 @@
...
+    <link href="./public/css/style.css" rel="stylesheet" />
+    <link href="./public/css/flexboxgrid.css" rel="stylesheet" />
+    <link href="./public/css/fontawesome.css" rel="stylesheet" />
+    <link href="./public/css/all.css" rel="stylesheet" />
+</head>
+<body>
+    <header>
+        <h1>Home Gallery - javascript-30</h1>
+        <br>
+        <p>This page contains a collection of javascript-30 project.
```
Comment (bermarte) [inline index.html:16]: isn't it 'projects'?

File: index.html [hunk 2]
```diff
@@ -1,25 +1,47 @@
...
+        The idea behind this project is to apply SoC (Separation of Concerns) concept.<br>
```
Comment (bermarte) [inline index.html:17]: If you write 'projects' on line 16, you can delete 'this project  on line 17.

File: index.html [hunk 3]
```diff
@@ -1,25 +1,47 @@
...
+              as little as possible."<br>
+              <a href="https://effectivesoftwaredesign.com/2012/02/05/separation-of-concerns/">Effective Software Design</a></p>
+    </header>
+    <section id="projects">
+        <div class="container">
+            <div class="row row center-xs center-sm center-md center-lg middle-lg middle-md middle-xs middle-sm features2">
+                <div class="col-xs-12 col-sm-3 col-md-3 col-lg-3">
+                <h3> Javascript Drum Kit</h3>
+                <img src="">
+                <a href="./JavaScript Drum Kit/index-FINISHED.html">link to first project</a>
```
Comment (bermarte) [inline index.html:29]: the actual link shouldn't contain empty spaces, I think.

File: index.html [hunk 4]
```diff
@@ -1,25 +1,47 @@
...
+             The idea that a software system must be decomposed into parts that overlap in functionality
+              as little as possible."<br>
+              <a href="https://effectivesoftwaredesign.com/2012/02/05/separation-of-concerns/">Effective Software Design</a></p>
+    </header>
+    <section id="projects">
+        <div class="container">
+            <div class="row row center-xs center-sm center-md center-lg middle-lg middle-md middle-xs middle-sm features2">
+                <div class="col-xs-12 col-sm-3 col-md-3 col-lg-3">
+                <h3> Javascript Drum Kit</h3>
+                <img src="">
```
Comment (bermarte) [inline index.html:28]: you should find some images and use the 'alt' attribute.

**Review by bermarte — COMMENTED** [review]
Just testing how reviews work.

**Comment by bermarte** [comment]
Let's see, I think I can merge it.

**Comment by bermarte** [comment]
They are just minor things.

**Pull request merged**